contrib/confluentinc/confluent-kafka-go/kafka/headers.go

```diff
@@ -8,7 +8,7 @@ package kafka
 import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 
-	"gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
```

Comment:
I see what the problem is here. Sorry for missing the point from the issue. What we need to do here is to continue supporting both, for older users. What we usually do in this case is apply this change and simply add a "new integration" at the `contrib/gopkg.in/confluentinc/confluent-kafka-go.v1/kafka` path.

Then we'll basically have two copies, each represented by the right path, for all users (old and new). We don't want to risk upsetting anyone.

Reply:
Got it!
For this case in terms of package naming maybe more suitable will be use `contrib/gopkg.in/confluentinc/confluent-kafka-go.v1/kafka` for old approach because package `gopkg.in/confluentinc/confluent-kafka-go.v1/kafka` is used there, and leave `contrib/gopkg.in/confluentinc/confluent-kafka-go/kafka` for the github one.

Reply:
Oh, right. I see what you mean. I was looking at things backwards. Sorry about that. My bad.

It seems like we have bad naming here. In this case, your change should be fine, without the need to introduce more confusion and copies.

One thing I'd like to make sure is that none of the older versions of the upstream library enforce [canonical import paths](https://golang.org/cmd/go/#hdr-Import_path_checking).

Reply:
As I saw confluent-kafka-go din't use any import comments in new nether old versions.